docs/content/docs/ops/metrics.md

```diff
@@ -1526,6 +1526,41 @@ Note that for failed checkpoints, metrics are updated on a best efforts basis an
 ### RocksDB
 Certain RocksDB native metrics are available but disabled by default, you can find full documentation [here]({{< ref "docs/deployment/config" >}}#rocksdb-native-metrics)
 
+### ForStDB
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 18%">Scope</th>
+      <th class="text-left" style="width: 26%">Metrics</th>
+      <th class="text-left" style="width: 48%">Description</th>
+      <th class="text-left" style="width: 8%">Type</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th rowspan="4"><strong>Task/Operator</strong></th>
+      <td>forst.fileCache.hit</td>
+      <td>The hit count of ForSt state backend cache.</td>
```

Comment:
I am curious.
Do the cache entries get invalidated at any time , due to size or time? Is this something we can capture in the metrics? I assume a miss will result in the value being added to the cache.
It would be useful to include in the documentation how to interpret the hit and miss counts and when they indicate that something needs to be changed / tuned.

Reply:
> Do the cache entries get invalidated at any time , due to size or time? 

Yes.

> Is this something we can capture in the metrics?

Something means what? could you give an example?

> Can the counters be reset - would this be useful?

The counter can not be reset while the job is running, this counter is a common counter defined in Flink, if you want to make the counter resettable, a discussion may need to be started.

Reply:
> > Do the cache entries get invalidated at any time , due to size or time?
> 
> Yes.
> 
> > Is this something we can capture in the metrics?
> 
> Something means what? could you give an example?
> 
> > Can the counters be reset - would this be useful?
> 
> The counter can not be reset while the job is running, this counter is a common counter defined in Flink, if you want to make the counter resettable, a discussion may need to be started.

@fredia thanks for your reply. Sorry for the bassc questions.  I am not sure how the invalidation occurs - can you count the number of invalidations in a similar way to the hits and misses.

Reply:
> I am not sure how the invalidation occurs - can you count the number of invalidations in a similar way to the hits and misses.

I think "invalidations" == "misses" for a cache. Take a fixed-sized  FIFO cache as example, when the requested data is not found in the cache, a "miss(or invalidations )" happens. Due to limited size of the cache, if the cache is full and new data needs to be added, the oldest data (first-in) will be replaced. If this removed data is needed again later, it results in a miss.